models/evc.py

```diff
@@ -465,6 +496,42 @@ def remove_current_flows(self, current_path=None, force=True):
         self.deactivate()
         self.sync()
 
+    def remove_path_flows(self, path=None, force=True):
+        """Remove all flows from path."""
+        if not path:
+            return
+
+        dpid_flows_match = {}
+        for dpid, flows in self._prepare_nni_flows(path).items():
+            dpid_flows_match.setdefault(dpid, [])
+            for flow in flows:
+                dpid_flows_match[dpid].append({
+                    "cookie": flow["cookie"],
+                    "match": flow["match"],
+                    "cookie_mask": 18446744073709551615
+                })
+        for dpid, flows in self._prepare_uni_flows(path, skip_in=True).items():
+            dpid_flows_match.setdefault(dpid, [])
+            for flow in flows:
+                dpid_flows_match[dpid].append({
+                    "cookie": flow["cookie"],
+                    "match": flow["match"],
+                    "cookie_mask": 18446744073709551615
```

Comment:
```suggestion
                    "cookie_mask": int(0xffffffffffffffff)
```

Sometimes, I find myself converting to hex to double check the mask, tiny detail, but since the code is read more times than it's written, having the hex maybe facilitate.  Maybe we could add this as a minor guideline, let me know if you also share the same impression when reading ints as base 10 or their hex values for the cookie_mask.

Reply:
yes, that is true!! I agree with you